plugins/techdocs/src/home/components/TechDocsCustomHome.tsx

```diff
@@ -124,7 +124,16 @@ export const TechDocsCustomHome = ({
   const configApi: ConfigApi = useApi(configApiRef);
 
   const { value: entities, loading, error } = useAsync(async () => {
-    const response = await catalogApi.getEntities();
+    const response = await catalogApi.getEntities({
+      filter: [
+        { kind: 'API' },
+        { kind: 'Component' },
+        { kind: 'Resource' },
+        { kind: 'System' },
+        { kind: 'Domain' },
```

Comment:
Ideally this would be replaced with a blacklist type filter in the future once it's supported so we just exclude kinds that doesn't make sense to have docs.